src/ansys/fluent/core/utils/quantity_config.json

```diff
@@ -0,0 +1,457 @@
+        },
+        "m": {
+            "type": "Length",
+            "factor": 1.0,
+            "offset": 0.0,
+            "dimension_order": 2 
+
+        },
+        "cm": {
+            "type": "Length",
+            "factor": 0.01,
+            "offset": 0.0,
+            "dimension_order": 2
+        },
+        "ft": {
+            "type": "Length",
+            "factor": 0.30479999999999996,
+            "offset": 0.0,
+            "dimension_order": 2
+        },
+        "in": {
+            "type": "Length",
+            "factor": 0.0254,
+            "offset": 0.0,
+            "dimension_order": 2
+        },
+        "s": {
+            "type": "Time",
+            "factor": 1.0,
+            "offset": 0.0,
+            "dimension_order": 3
+        },
+        "A": {
+            "type": "Current",
+            "factor": 1.0,
+            "offset": 0.0,
+            "dimension_order": 8
+        },
+        "mol": {
+            "type": "Chemical Amount",
+            "factor": 1.0,
+            "offset": 0.0,
+            "dimension_order": 6
+        },
+        "slugmol": {
+            "type": "Chemical Amount",
+            "factor": 14593.9,
+            "offset": 0.0,
+            "dimension_order": 6
+        },
+        "cd": {
+            "type": "Light",
+            "factor": 1.0,
+            "offset": 0.0,
+            "dimension_order": 7
+        },
+        "sr": {
+            "type": "Solid Angle",
+            "factor": 1.0,
+            "offset": 0.0,
+            "dimension_order": 9
+        },
+        "radian": {
+            "type": "Angle",
+            "factor": 1.0,
+            "offset": 0.0,
+            "dimension_order": 5
+        },
+        "degree": {
+            "type": "Angle",
+            "factor": 0.017453292519943295,
+            "offset": 0.0,
+            "dimension_order": 5
+        },
+        "K": {
+            "type": "Temperature",
+            "factor": 1.0,
+            "offset": 0.0,
+            "dimension_order": 4
+        },
+        "C": {
+            "type": "Temperature",
+            "factor": 1.0,
+            "offset": 273.15,
+            "dimension_order": 4
+        },
+        "F": {
+            "type": "Temperature",
+            "factor": 0.55555555555,
+            "offset": 459.67,
+            "dimension_order": 4
+        },
+        "R": {
+            "type": "Temperature",
+            "factor": 0.55555555555,
+            "offset": 0.0,
+            "dimension_order": 4
+        },
+        "delta_K": {
+            "type": "Temperature Difference",
+            "factor": 1.0,
+            "offset": 0.0,
+            "dimension_order": 4
+        },
+        "delta_C": {
+            "type": "Temperature Difference",
+            "factor": 1.0,
+            "offset": 0.0,
+            "dimension_order": 4
+        },
+        "delta_F": {
+            "type": "Temperature Difference",
+            "factor": 1.0,
```

Comment:
Should that not be the factor to convert to `delta_K` or `delta_C`?

Maybe you can throw a comment at the top which covers what each entry is for.

Reply:
https://github.com/ansys/pyfluent/blob/f3f6e135ef9804f9c563c2ae66eb4c9154ff8a2e/src/ansys/fluent/core/utils/quantity_config.yaml#L1-L9

Fixed the factors for delta_F, delta_C, and delta_R. Also switched from json to yaml to include comments in the config file explaining the properties within each unit.